src/components/CookieStatement.vue

```diff
@@ -126,3 +126,9 @@
     </p>
   </div>
 </template>
+
+<style lang="scss">
+@import '../tos-styles.scss'
```

Comment:
Please fix the warnings. You can view them on Github: https://github.com/UrbanSystemsLab/climateiq-login/pull/25/files

They may also be fixed by autoformatting: `npm run lint -- --fix`

Reply:
Resolved!